tests/mir-opt/dest-prop/copy_propagation_arg.rs

```diff
@@ -9,25 +8,42 @@ fn dummy(x: u8) -> u8 {
 
 // EMIT_MIR copy_propagation_arg.foo.DestinationPropagation.diff
 fn foo(mut x: u8) {
+    // CHECK-LABEL: fn foo(
+    // CHECK: debug x => [[x:_.*]];
+    // CHECK: dummy(move [[x]])
+    // CHECK: [[x]] = move {{_.*}};
     // calling `dummy` to make a use of `x` that copyprop cannot eliminate
     x = dummy(x); // this will assign a local to `x`
 }
 
 // EMIT_MIR copy_propagation_arg.bar.DestinationPropagation.diff
 fn bar(mut x: u8) {
+    // CHECK-LABEL: fn bar(
+    // CHECK: debug x => [[x:_.*]];
+    // CHECK: dummy(move [[x]])
+    // CHECK: [[x]] = const 5_u8;
     dummy(x);
     x = 5;
 }
 
 // EMIT_MIR copy_propagation_arg.baz.DestinationPropagation.diff
 fn baz(mut x: i32) -> i32 {
+    // CHECK-LABEL: fn baz(
+    // CHECK: debug x => [[x:_.*]];
+    // CHECK-NOT: [[x]] = {{_.*}}
     // self-assignment to a function argument should be eliminated
     x = x;
     x
 }
 
 // EMIT_MIR copy_propagation_arg.arg_src.DestinationPropagation.diff
 fn arg_src(mut x: i32) -> i32 {
+    // CHECK-LABEL: fn arg_src(
+    // CHECK: debug x => [[x:_.*]];
+    // CHECK: debug y => [[y:_.*]];
+    // CHECK: [[y]] = [[x]]
+    // CHECK: [[x]] = const 123_i32;
+    // CHECK-NOT: {{_.*}} = [[y]];
```

Comment:
Should we directly give them their names, aka `x` is _1 and `y` is _0?

Reply:
Hmm for me, it feels easier to read the test because you can more easily read the checks directly referencing the variable names (x and y), but maybe I'm misunderstanding something.